hydroserving/core/model/entities.py

```diff
@@ -11,18 +12,22 @@ class VersionStatus(Enum):
 
 
 class UploadMetadata:
-    def __init__(self, name, contract, host_selector, runtime, install_command, metadata):
+    def __init__(self, name, contract, host_selector, runtime, install_command, metadata, monitoring_configuration: MonitoringConfiguration):
         self.contract = contract
         self.hostSelectorName = host_selector
         self.runtime = runtime
         self.name = name
         self.installCommand = install_command
         self.metadata = metadata
 
+        if not monitoring_configuration:
+            monitoring_configuration = MonitoringConfiguration()
+        self.monitoring_configuration = monitoring_configuration.__dict__
```

Comment:
why `__dict__` ?

Reply:
Otherwise it should be converted each time when creating UploadMetaData object. Or do you think that it is the desired way? ( I saw that conversion is done manually during creating obj as in https://github.com/Hydrospheredata/hydro-serving-cli/blob/8f8587c93c33f66e08fd4cc653087c92507b649c/hydroserving/core/model/upload.py)

Reply:
Convert to dict only when you need to create a json payload. 
In every other case you need to keep the types clean and explicit.

Reply:
Fixed